emulate-something.ts

```diff
@@ -0,0 +1,63 @@
+import {
+  Emulator,
+  fromText,
+  generatePrivateKey,
+  getAddressDetails,
+  Lucid,
+  toUnit,
+  TxHash,
+} from "../../mod.ts";
+
+const privateKey = generatePrivateKey();
+
+const address = await (await Lucid.new(undefined, "Custom"))
+  .selectWalletFromPrivateKey(privateKey).wallet.address();
+
+const { paymentCredential } = getAddressDetails(address);
+
+const emulator = new Emulator([{ address, assets: { lovelace: 3000000000n } }]);
+
+const lucid = await Lucid.new(emulator);
```

Comment:
Emulator is a class; 
`new` means instantiation.

N.B. `lucid` vs `Lucid`

https://deno.land/x/lucid@0.10.1/mod.ts?s=Lucid&p=new
https://deno.land/x/lucid@0.10.1/mod.ts?s=Emulator